var/spack/repos/builtin/packages/lmod/package.py

```diff
@@ -51,7 +51,7 @@ class Lmod(AutotoolsPackage):
 
     def setup_environment(self, spack_env, run_env):
         stage_lua_path = join_path(
-            self.stage.path, 'Lmod-{version}', 'src', '?.lua')
+            self.stage.source_path, 'Lmod-{version}', 'src', '?.lua')
```

Comment:
This one should be `self.stage.source_path, 'src', '?.lua')`, as `Lmod-<version>` is now just `src`

Reply:
Thanks.  I'm actually testing that fix now.